src/crystal/system/unix/file_info.cr

```diff
@@ -54,7 +54,7 @@ struct Crystal::System::FileInfo < ::File::Info
     @stat.st_gid.to_u32
   end
 
-  def ==(other : ::File::Info) : Bool
+  def same_file?(other : ::File::Info) : Bool
```

Comment:
This now needs an `abstract def` in `src/file/info.cr`.

Reply:
Updated in the next commit